packages/engine/src/editor/nodes/ReflectionProbeNode.ts

```diff
@@ -14,6 +10,8 @@ import {
   Vector3,
   WebGLCubeRenderTarget
 } from 'three'
+import { FilesToUpload } from '../../../../client-core/src/world/components/editor/Api'
```

Comment:
we must keep the package hierarchy in fact, the engine cannot depend on client-core, only client-core on the engine, so i think these interfaces should be moved to the engine. we will be overhauling the editor next month anyway, so this will all change again, but for now we must preserve this so npm distribution works.